src/grouping.jl

```diff
@@ -22,8 +22,8 @@ julia> schema((; grp = string.(1:100_000)), Dict(:grp => Grouping()))
 struct Grouping <: StatsModels.AbstractContrasts end
 
 # return an empty matrix
-StatsModels.contrasts_matrix(::Grouping, baseind, n) = zeros(0, 0)
-StatsModels.termnames(::Grouping, levels::AbstractVector, baseind::Integer) = levels
+# StatsModels.contrasts_matrix(::Grouping, baseind, n) = error("Grouping terms don't have associated contrasts")
+# StatsModels.termnames(::Grouping, levels::AbstractVector, baseind::Integer) = levels
```

Comment:
probably better to just clean up this cruft, OR leave the methods as they were before with a note that they're _probably_ not needed since we've added a direct constructor for ContrastsMatrix